wallet/src/actors/app/handlers/get_addresses.rs

```diff
@@ -28,7 +28,7 @@ impl Handler<GetAddressesRequest> for app::App {
             .offset
             .unwrap_or_else(|| constants::DEFAULT_PAGINATION_OFFSET);
         let limit = cmp::min(
-            msg.offset
+            msg.limit
                 .unwrap_or_else(|| constants::DEFAULT_PAGINATION_LIMIT),
             constants::MAX_PAGINATION_LIMIT,
```

Comment:
Why keep the limit? :smiley:

Reply:
To use a similar logic as in `get_transactions` :laughing:

Reply:
I removed it along with the one in `get_transactions`